src/mlpack/methods/cf/cf_model.hpp

```diff
@@ -24,6 +25,13 @@
 #include <mlpack/methods/cf/decomposition_policies/bias_svd_method.hpp>
 #include <mlpack/methods/cf/decomposition_policies/svdplusplus_method.hpp>
 
+#include <mlpack/methods/cf/normalization/no_normalization.hpp>
+#include <mlpack/methods/cf/normalization/overall_mean_normalization.hpp>
+#include <mlpack/methods/cf/normalization/user_mean_normalization.hpp>
+#include <mlpack/methods/cf/normalization/item_mean_normalization.hpp>
+#include <mlpack/methods/cf/normalization/z_score_normalization.hpp>
+#include <mlpack/methods/cf/normalization/combined_normalization.hpp>
```

Comment:
No need to include combined_normalization.hpp here since it is not supported yet.

Reply:
Oh yes, thanks!